### PR TITLE
Add fetch missions and render missions

### PR DESCRIPTION
**Pull request by Ewoskyx**
### FETCH MISSIONS & DISPLAY

Hello, In this update, I did;
- Add fetch missions
- Add fetch display

### CODE REVIEW

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Changes Required ♻️

Hi @Ewoskyx , your project feature is looking great and you're almost at the end of completing it. 
There are a few changes I want you to look into:
- In your Missions.JSX, it would be nice you strip out the table head into a child component and call it in the Missions.JSX component. As that will make your component lean and cleaner.

File: src/components/Missions/Missions.jsx [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+                      Description
+                    </th>
+                    <th scope="col" className="text-sm font-medium text-gray-900 px-6 py-4 text-left">
+                      Status
+                    </th>
+                    <th scope="col" className="text-sm font-medium text-gray-900 px-6 py-4 text-left">
+                      {' '}
+                    </th>
+                  </tr>
+                </thead>
```
Comment (AmaduKamara) [inline src/components/Missions/Missions.jsx]: It would be nice you strip this out into a child component and then call it in your Missions.JSX component

**Review by AmaduKamara — CHANGES_REQUESTED** [review]

File: src/components/Missions/Missions.jsx (hunk 1 above)
Comment (Ewoskyx) [reply src/components/Missions/Missions.jsx]: Thank you @AmaduKamara , I  will do the changes :)

**Review by AmaduKamara — APPROVED** [review]
# Project Approved 💪

Congratulations @Ewoskyx , your project's feature is complete and there is nothing more to say other than it's time to merge your project.

Happy coding 👨‍💻👨‍💻👨‍💻

**Pull request merged**